client/styles/components/component_kit/cw_text_input.scss

```diff
@@ -0,0 +1,60 @@
+@import 'client/styles/facelift';
+
+.TextInput {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+
+  label {
+    @include form-label;
+    margin-bottom: 4px;
+  }
+
+  input {
+    border-radius: 4px;
+    border: 2px solid $white;
+    background-color: $background-gray;
+    height: 42px;
+    padding: 8px 15px;
+
+    &:hover {
+      border: 2px solid $lite-purp;
+    }
+
+    &:focus {
+      outline: none;
+    }
+
+    &:focus-within {
+      border: 2px solid $midi-purp;
```

Comment:
we have "border: 2px solid" quite a bit, can we make a css variable for "textinputborder" that's called here instead? 

keeping in mind, having a single scss file full of variables we can tweak in the future and watch style trickle down